requirements_tests.txt

```diff
@@ -1,21 +1,13 @@
-six
-sqlalchemy
--e git+https://github.com/ODM2/geoalchemy.git@odm2
-#Change line #242 in geoalchemy/base.py
-#from:
-#class SpatialComparator(ColumnProperty.ColumnComparator):
-#to :
-#class SpatialComparator(ColumnProperty.Comparator):
-
-shapely
-
-#matplotlib
-dateutils
-pandas
-
+-r requirements.txt
+pytest
+pytest-cov
+mock
 #db support
 pymysql
 pysqlite
-pyspatialite
+pymssql
+psycopg2
+#pyspatialite
```

Comment:
> I would remove it rather than comment it out.

That applies to most of the code commented out in this PR BTW. Let git and the PR be the tracker of the changes :wink: